include/utility/dbus_utility.hpp

```diff
@@ -366,5 +366,33 @@ inline bool isChassisPowerOn()
     */
     return false;
 }
+
+/**
+ * @brief API to check if host is in running state.
+ *
+ * This API reads the current host state from D-bus and returns true if the host
+ * is running.
+ *
+ * @return true if host is in running state. false otherwise.
+ */
+inline bool isHostRunning()
+{
+    auto l_hostState = dbusUtility::readDbusProperty(
+        "xyz.openbmc_project.State.Host", "/xyz/openbmc_project/state/host0",
```

Comment:
should we move these to constants ?

Reply:
Jinu: use the Host service name from constants::